final_prep/README.md

```diff
@@ -44,31 +44,32 @@ In Mod 0 you've learned about different techniques for managing your time at Tur
 
 When you are finished, add screenshots of your calendar so we can provide feedback if needed!
 
-- `Add Week 1 Screenshot Here`
-- `Add Week 2 Screenshot Here`
-- `Add Week 3 Screenshot Here`
+- `Add Week 1 Screenshot Here` <img width="1204" alt="Screen Shot 2022-02-12 at 1 17 58 PM" src="https://user-images.githubusercontent.com/98674727/153728804-0763b70e-9c59-462f-8e9f-701d4a08d075.png">
+- `Add Week 2 Screenshot Here`<img width="1209" alt="Screen Shot 2022-02-12 at 1 19 23 PM" src="https://user-images.githubusercontent.com/98674727/153728814-adc440d2-20e6-4c52-94b1-c29ab14118a7.png">
+- `Add Week 3 Screenshot Here` <img width="1211" alt="Screen Shot 2022-02-12 at 1 19 36 PM" src="https://user-images.githubusercontent.com/98674727/153728829-51220745-119c-493a-a454-9283b150afc0.png">
```

Comment:
Calendar is looking great. I know the Mod 1 calendar hasn't been fully updated yet but when it does, highly recommend specifying some of your work blocks (ie, what will you study based on the lessons that week?).

Big fan of how you're keeping `zone out mindless scroll time` on the schedule, but timeboxing it too.